### PR TITLE
Run the call centre tests

### PR DESCRIPTION
**Pull request by rowlando**
## What does this pull request do?

It runs the call centre tests. Well, the ones that are actually passing 😕. I've also added an instruction to the README that says how to run the tests:

```
$ bin/setup && bin/run_tests && bin/stop
```

### Flaky test
One of the tests might be flaky. @aliuk2012 is working on the tests, which currently reside in https://github.com/ministryofjustice/cla_frontend/pull/620.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ services:
 
   # applications
   cla_public:
-    image: ${ECR_ENDPOINT}/cla_public:develop
+    image: 627373361525.dkr.ecr.eu-west-1.amazonaws.com/get-access/cla_public:develop
```
Comment (sldblog) [inline docker-compose.yaml]: 👏

File: docker-compose.yaml [hunk 2]
```diff
@@ -75,7 +75,7 @@ services:
     depends_on:
       - cla_backend
   cla_backend:
-    image: ${ECR_ENDPOINT}/cla_backend:develop
+    image: 627373361525.dkr.ecr.eu-west-1.amazonaws.com/get-access/cla_backend:develop
```
Comment (sldblog) [inline docker-compose.yaml]: 👏

File: tests/call-centre/protractor.conf.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ exports.config = {
       binary: puppeteer.executablePath()
     }
   },
-  baseUrl: "http://django/",
+  baseUrl: "http://cla_frontend/",
```
Comment (sldblog) [inline tests/call-centre/protractor.conf.js]: _not a blocker_

💭 I imagine at some point we will have to run these outside the docker-compose network context, where we will have to configure the URLs via `localhost:8001`, etc.

File: bin/stop [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+docker-compose stop
```
Comment (sldblog) [inline bin/stop]: 🙋‍♂️ We could use `down` instead if the aim is to clean up as `stop` does not remove the containers.

File: README.md [hunk 5]
```diff
@@ -86,11 +86,11 @@ The browser tests use these technologies:
...
 1. Run `aws ecr describe-repositories`. You should see a list of ECR repositories.
 
-#### Run the scripts
+#### Run the tests
+The following command will (1) start services and applications, (2) run the tests and (3) stop the services and applications.
 
 ```
-$ bin/setup
-$ bin/run
+$ bin/setup && bin/run_tests && bin/stop
```
Comment (sldblog) [inline README.md]: 🙋‍♂️ Should we include a CircleCI job to do this as part of this pull request?

**Comment by sldblog** [comment]
> One of the tests might be flaky

Should we mark it ignored?

File: tests/call-centre/protractor.conf.js (hunk 3 above)
Comment (rowlando) [reply tests/call-centre/protractor.conf.js]: :thumbsup:

File: README.md (hunk 5 above)
Comment (rowlando) [reply README.md]: I'll do that in a separate pull request

File: bin/stop (hunk 4 above)
Comment (rowlando) [reply bin/stop]: :thumbsup:

**Pull request merged**